science/orthanc-dicomweb/Makefile

```diff
@@ -0,0 +1,64 @@
+# Created by: maintainer.freebsd@xpoundit.com
+
+PORTNAME=	orthanc-dicomweb
+DISTVERSION=	1.6
+CATEGORIES=	science
+MASTER_SITES=	https://www.orthanc-server.com/downloads/get.php?path=/plugin-dicom-web/:main \
+		https://www.orthanc-server.com/downloads/get.php?path=/orthanc/:framework \
+		https://orthanc.osimis.io/ThirdPartyDownloads/:thirdparty \
+		https://orthanc.osimis.io/ThirdPartyDownloads/dicom-web/:thirdpartydicomweb
+DISTFILES=	OrthancDicomWeb-${PORTVERSION}.tar.gz:main \
+		Orthanc-1.9.3.tar.gz:framework \
+		e2fsprogs-1.44.5.tar.gz:thirdparty \
+		bootstrap-4.3.1.zip:thirdpartydicomweb \
+		vuejs-2.6.10.tar.gz:thirdpartydicomweb \
+		axios-0.19.0.tar.gz:thirdpartydicomweb \
+		Font-Awesome-4.7.0.tar.gz:thirdpartydicomweb \
+		bootstrap-vue-2.0.0-rc.24-dist.tar.gz:thirdpartydicomweb \
+		babel-polyfill-6.26.0.min.js.gz:thirdpartydicomweb
+DIST_SUBDIR=	orthanc
+EXTRACT_ONLY=	OrthancDicomWeb-${PORTVERSION}.tar.gz
+
+MAINTAINER=	maintainer.freebsd@xpoundit.com
+COMMENT=	Plugin to bring support of the DICOMweb standard into Orthanc
```

Comment:
Just nitpicking, but hope those COMMENTs could align, now they have:
Orthanc plugin
Plugin to bring support ... into Orthanc
Plugin to extend Orthanc

Reply:
Nice catch, didn't really pay too much attention to COMMENTs across this ports - should have done.